packages/metrics/tests/unit/middleware/middy.test.ts

```diff
@@ -247,14 +247,14 @@ describe('Middy middleware', () => {
             CloudWatchMetrics: [
               {
                 Namespace: 'serverlessAirline',
-                Dimensions: [[ 'environment', 'aws_region', 'service', 'function_name' ]],
+                Dimensions: [[ 'service', 'environment', 'aws_region', 'function_name' ]],
```

Comment:
This change (& the one a few lines below) is needed because we are comparing the `JSON.stringif`-ied version of these strings, so order counts.

I could have converted to object comparison but wanted to keep the diff small.